gpkit/geometric_program.py

```diff
@@ -43,7 +43,8 @@ def __init__(self, cost, constraints, verbosity=1):
         self.cost = cost
         self.constraints = constraints
         self.posynomials = [cost] + list(constraints)
-        NomialData.__init__(self, signomials=self.posynomials, simplify=False)
+        super(GeometricProgram, self).__init__(signomials=self.posynomials,
```

Comment:
This seems harder to read and less correct. We want to create a NomialData of the current object, not generic `super`